### PR TITLE
Feature adding page style

### PR DESCRIPTION
**Pull request by OmarMWarraich**
Page and assets added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="body-wrapper">
+        <nav class="header-flex">
+            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
+            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
+        </nav>
```
Comment (Bondok6) [inline index.html:16]: - [x] I suggest using HTML5 semantic tags (e.g., header, nav) over divs to Follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), so the main container carrying the header section should be a `<header>` tag. kindly fix that. 👌

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <div class="container-flex">
+            <div class="intro-flex">
+                <h1>Omar Mohammad</h1>  
+                <h2>Software Developer</h2>        
+            </div>
```
Comment (Bondok6) [inline index.html:21]: - [x] Great job using `<h1>` tag here 💯, but remember according to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, change the font color and add horizontal margin between these elements to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        <br>
+            <div class="about-flex">
+                <p class="about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
```
Comment (Bondok6) [inline index.html:25]: - [ ] the same thing here, please change the font color for this paragraph to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

> the font color should be `color: #C8C8D4;`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        </div>
+        
+
+                <div class="media-flex">
+                    <a href="#"><img alt= 'git' src="./assets/Github.svg"></a>
+                    <a href="#"><img alt= 'Vic' src="./assets/Victory.svg"></a>
+                    <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
+                    <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
+                </div>
+
```
Comment (Bondok6) [inline index.html:70]: - [x] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, Change the `<h4>` font color, Add some horizontal spaces between the elements to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            <hr>
+        </div>
+        
+
+                <div class="media-flex">
+                    <a href="#"><img alt= 'git' src="./assets/Github.svg"></a>
+                    <a href="#"><img alt= 'Vic' src="./assets/Victory.svg"></a>
+                    <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
+                    <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
+                </div>
```
Comment (Bondok6) [inline index.html:69]: - [x] I suggest using `<ul>` and `<li>` tags here 👌.  the list of icons should be grouped by a `<ul>` tag and each `<a>` tag should be in `<li>` tag.

> For example
```
<ul class="media-flex">
  <li><a href="#"><img alt= 'git' src="./assets/Github.svg"></a></li>
  <li><a href="#"><img alt= 'Vic' src="./assets/Victory.svg"></a></li>
  <li><a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a></li>
  <li><a href="#"><img alt= 'M' src="./assets/M.svg"></a></li>
</ul>
```

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @OmarMWarraich 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- You've used the correct Github flow ✅
- All linters are passing ✅
- Well structured files ✅

## Required Changes ♻️

> #### ⭕ Important Note: Please update your summary of the PR and write a descriptive and short summary of changes introduced in the pull request.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (OmarMWarraich) [reply index.html:16]: Noted and resolved. Thanks a lot.

File: index.html (hunk 2 above)
Comment (OmarMWarraich) [reply index.html:21]: OK.

File: index.html (hunk 5 above)
Comment (OmarMWarraich) [reply index.html:69]: Well, it really sums this up.

File: index.html (hunk 4 above)
Comment (OmarMWarraich) [reply index.html:70]: Sorry,..my bad.

**Comment by OmarMWarraich** [comment]
Some thoughtful changes added at the reviewer's insight.

**Review by Bondok6 — COMMENTED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @OmarMWarraich 🙋‍♂️, 

> ### ⭕ Important Note: Please don't merge or close your pull request before getting approval.

Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great work on making the changes requested ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
-  Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**